src/main/java/com/ekart/config/SecurityConfig.java

```diff
@@ -58,4 +68,17 @@ public AuthenticationManager authenticationManagerBean() throws Exception {
         return super.authenticationManagerBean();
     }
 
+    @Bean
+    CorsConfigurationSource corsConfigurationSource() {
+        CorsConfiguration configuration = new CorsConfiguration();
```

Comment:
All these can be configured in corsfilter itself

Reply:
Spring security's filters have higher order as compared to other filters. Because of that, it will execute before the cors filter.
Our cors filter will not execute before the security filter. So if we want to config anything which spring security needs will be done in this file.